posts/MarkoB/2017-02-06-validator-service-loader.adoc

```diff
@@ -4,21 +4,20 @@ Marko Bekhta
 :awestruct-layout: blog-post
 ---
 
-Have you ever been in a situation, when you would like to get even more from the lib
+Have you ever been in a situation when you would like to get even more from the library
 that you are using all the time? Or when you want to somehow change its behavior ?
 
-Today we'll be talking about Hibernate Validator and how you can provide your own constraint
-and/or validator in a fully self-contained manner. Meaning packaging it all into it's own jar file,
+Today we'll be talking about Hibernate Validator and how you can provide your own constraints
+and/or validators in a fully self-contained manner. Meaning packaging it all into its own jar file,
 in a way that the others can use your lib for their needs.
 
-This functionality is based on Hibernate Validators usage of Java's https://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html[ServiceLoader]
-mechanism that allows to register additional constraint definitions
-(https://docs.jboss.org/hibernate/stable/validator/reference/en-US/html_single/#_constraint_definitions_via_code_serviceloader_code[see here]).
+This functionality is based on Hibernate Validator usage of Java's https://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html[ServiceLoader]
+mechanism that allows to https://docs.jboss.org/hibernate/stable/validator/reference/en-US/html_single/#_constraint_definitions_via_code_serviceloader_code[register additional constraint definitions].
```

Comment:
Did you check that the link is working correctly? GitHub interprets the underscores as italic so I'd like to be sure it's OK.